ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/radius_server.schema.yml

```
@@ -0,0 +1,48 @@
+type: dict
+keys:
+  attribute_32_include_in_access_req:
+    type: dict
+    keys:
+      hostname:
+        type: bool
+      format:
+        type: str
+        description: Name of ACL
```

Are you sure this is correct?

I would expect some static valid values here.

---

Yes it is correct. If hostname is true will add "hostname" in command string. 
{%         if radius_server.attribute_32_include_in_access_req.hostname is arista.avd.defined (true) %}
{%             set attribute_32_include_in_access_cli = attribute_32_include_in_access_cli ~ " hostname" %}
{%         elif radius_server.attribute_32_include_in_access_req.format is arista.avd.defined %}
{%             set attribute_32_include_in_access_cli = attribute_32_include_in_access_cli ~ " format " ~ radius_server.attribute_32_include_in_access.format %}
{%         endif %}

---

I checked some documentation and EOS CLI and the `format` should not be `Name of ACL`.
```suggestion
        description: Specify the format of the NAS-Identifier. If 'hostname' is set, this is ignored.
```